### DISCUSSION TITLE
The program won't open

### DISCUSSION THREAD

**Discussion by Wilson-Smith** [Monika-After-Story/MonikaModDev#9205] [QA - tech support + general]
So, today I wanted to gift to Monika a lot of stuff, but after I said 'goodbye', load all the gifts, the program didn't open.
I don't know because I load only sprites and gifts, nothing else.
But I have to say that Monika's folder it is about 1.14 GB, so MAYBE can't load such big files.
I don't really know what's going on, but fortunately I have a backup.
The thing is, why it don't want to open?
(It's useless put foto of a failed load, because yeah I double-click on it, there is the loading blue circle and after that the app should run, but now didn't run)

**Comment by Wingdinggaster656** [comment]
You mean you double-clicked the `.exe`, then nothing opened?

The reasons are varied. I know a lot of people have experienced this with MAS, and I know a lot of people have experienced this with processes other than MAS.

This seems less like a problem with the computer itself than with the program.
It could be running out of memory or something else. 99.999% of the time, just restarting the computer will fix everything.

EDIT: A reason I say this is a computer problem: all of the cases I've heard of have occurred on computers that aren't too new.

**Comment by Wilson-Smith** [comment]
Fortunately I had a backup of the Monika's folder (not the persistent folder) on my drive and I loaded it and it opened.
I think the .exe file won't open because I loaded too many sprite packs and too many gifts.
I don't know what happened, but I can open the program.

**Comment by Justformas** [comment]
>So, today I wanted to gift to Monika a lot of stuff, but after I said 'goodbye', load all the gifts, the program didn't open.

> I think the .exe file won't open because I loaded too many sprite packs and too many gifts.

You put all the gift files in the "characters" folder at the same time? How many gifts was that? Aside from how long that might take Monika to go through them all, maybe it could cause a computer problem if all the memory is getting used or something.
I usually only give Monika one or a few gifts at one time and haven't had a problem like that when gifting. So you could try doing that.

**Comment by Wilson-Smith** [comment]
Well, more than 20...
Because yesterday was a special day

**Comment by Justformas** [comment]
> Well, more than 20... Because yesterday was a special day

You can still give her lots of stuff on one day by gifting one at a time, then telling Monika you're going to restart, and repeat. Actually I'm wondering if that might get you more affection too, because if you gift multiple things at once then some of the gifts don't always seem to give affection. Maybe they still might not always give affection even if you do that, but outside of special days you can just spread the gifts out over more days if you have the patience for that.

**Comment by ThePotatoGuy** [comment]
Startup could be slow if there were a lot of gifts, but it shouldn't take too long. You can also gift while the game is running, it'll take a minute or 2 before she recognizes them.
